### ISSUE TITLE
Export with differences between movies / series

### ISSUE THREAD

**Issue by neerfix** [daltonmenezes/netflix-list-exporter#2]
# Export with differences between movies / series

Now we can export all list, 👍🏼 list and 👎🏼 list. 
But would it be possible to export the list with only the series or movies ?

**Comment by daltonmenezes** [comment]
Hi @Neerfix , great suggestion. I'll check if it's possible, thanks.

**Comment by daltonmenezes** [comment]
@Neerfix Good news, yeah, it's possible. I'm working right now in the v3.00. I'm rewriting the extension from the scratch and I'll also include the possibility to copy the list of entire watched (including not rated ones) movies and series.

**Comment by neerfix** [comment]
@daltonmenezes Yeah ! It's fantastic, I'm impatient 😁

**Comment by daltonmenezes** [comment]
@Neerfix I'm sorry for the long delay, but I can't handle these features properly because the Netflix API has many complications / limitations to make this happen. I did the job, but the cost is slow as hell because I can't make many requests to the netflix API without breaking things and the workaround was to limit the number of requests per time and it takes longer than feasible for release it and even with this workaround, new problems were found (movies / series that have ratings, but are outside the netflix catalog). For now, I can't launch something like this, but I hope it will change in the near future.

in short, all of this is not possible because the endpoint of the ratings api does not differentiate what is a series or movie, the other end of the api that provides something like that with some tricks becomes unviable because it breaks everything when trying to deal with items that are no longer in the catalog or due to api request limitations.

I'm going to close this issue for now, but if that becomes possible I will open it again.
